ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -119,6 +119,15 @@ boolean addLocalTransaction(final Transaction transaction) {
     return addTransaction;
   }
 
+  void evictOldTransactions(Instant maximumRetentionPeriod) {
```

Comment:
maximumRetentionPeriod seems like a misleading name - this would have to be the earliestAllowedTransaction or should be a `Duration` which is subtracted from the current time in this method.